app.yaml

```diff
@@ -11,10 +11,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-runtime: custom
+runtime: ruby
 vm: true
 api_version: 1
 module: default
+# [START entrypoint]
+entrypoint: bundle exec foreman start --formation "$FORMATION"
+# [END entrypoint]
 
 resources:
```

Comment:
Also, for entrypoint: Including it avoids an extra question on deployment. Which seems to suggest it should be included.

```
Please enter the command to run this Ruby app in production, or leave 
blank to accept the default:
[bundle exec rackup -p 8080 -E production /app/config.ru] 

To avoid being asked for an entrypoint in the future, please add it to your app.yaml. e.g.
  entrypoint: bundle exec rackup -p 8080 -E production /app/config.ru
```